modules/apps/object/object-test/src/testFunctional/tests/SystemObjectAccount.testcase

```diff
@@ -62,12 +123,93 @@ definition {
 	}
 
 	@description = "LPS-151765 - Verify if Cascade deletion type of Relationship One to Many can delete and you also will delete the child related entry (if you also have permissions for it)"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanDeleteRelationshipWhenDeletionTypeIsCascade {
+		Accounts.openAccountsAdmin();
 
-		// TODO LPS-153534 CanDeleteRelationshipWhenDeletionTypeIsCascade pending implementation
+		Accounts.addBusinessAccount(
+			accountDescription = "Accounts Name Description for Test",
+			accountName = "Accounts for Test");
+
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldBusinessType = "Text",
+			fieldLabelName = "Custom Field",
+			fieldName = "customField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			deletionType = "cascade",
+			objectName_1 = "CustomObject",
+			objectName_2 = "AccountEntry",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.addObjectLayoutViaAPI(
+			layoutName = "Layout Test",
+			objectName = "CustomObject");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+		ObjectAdmin.goToLayoutsTab();
+
+		ObjectAdmin.selectSpecificKebabMenuOption(
+			kebabOption = "View",
+			objectLabel = "Layout Test");
+
+		ObjectAdmin.markLayoutAsDefault();
+
+		ObjectAdmin.goToLayoutTabOnLayouts();
+
+		ObjectAdmin.addTabFieldsOnLayout(
+			blockName = "Block 1",
+			columnsNumber = "1",
+			fieldLabelName = "Custom Field",
+			tabName = "Field Tab");
+
+		ObjectAdmin.addTabRelationshipsOnLayout(
+			relationshipsLabelName = "Relationship",
+			tabName = "Relationship Tab");
+
+		Button.clickSave();
+
+		Navigator.openURL();
```

Comment:
See https://github.com/timpak/liferay-portal/pull/246/commits/bf14ad0a80f987a6c1eefd12326ee5256059fdc5.

There shouldn't be any need to go to the home page before running the next three macros. Two macros are calling APIs while the third is navigating to a specific URL.

Reply:
@evanilson-santana FYI